src/framework/preferences/css/stylus/utils/Themes.styl

```diff
@@ -1,5 +1,6 @@
 // Contains utility functions for building contrast themes
 // !important is used for all styles to ensure that Preferences Framework's enactors are able to override the pages default styles.
+// Deprecated functions to separatedPanelPrefsEditor, separatedPanelPrefsEditorFrame, prefsEditor
```

Comment:
It's probably better to add a comment above each of the deprecated files, as it's more likely that it will be seen in that case.